edi/models/edi_exchange_consumer_mixin.py

```diff
@@ -45,3 +45,30 @@ def action_view_edi_records(self):
 
     def _edi_generate_records(self):
         raise NotImplementedError()
+
+    @api.model
+    def get_edi_access(self, doc_ids, operation, model_name=False):
+        """Retrieve access policy.
+        The behavior is similar to `mail.thread` and `mail.message`
+        and it relies on the access rules defines on the related record.
+        The behavior can be customized on the related model
+        by defining `_edi_exchange_record_access`.
+
+        By default `write`, otherwise the custom permission is returned.
+        """
+        DocModel = self.env[model_name] if model_name else self
+        create_allow = getattr(DocModel, "_edi_post_access", "write")
```

Comment:
_edi_post_access shouldn't be `_edi_exchange_record_access`?

Reply:
True, I forgot :facepalm: